test/test-udp-send-hang-loop.c

```diff
@@ -0,0 +1,98 @@
+/* Copyright (c) 2017 Trevor Norris <trev.norris@gmail.com>.
```

Comment:
can you assign this to "libuv project and contributors" instead? that's how we've been doing it lately.